pkg/aspect/root/flags/bazel_flags.go

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2022 Aspect Build Systems, Inc.
+ *
+ * Licensed under the aspect.build Commercial License (the "License");
+ * you may not use this file except in compliance with the License.
+ * Full License text is in the LICENSE file included in the root of this repository.
+ */
+
+/*
+Copyright © 2021 Aspect Build Systems Inc
+
+Not licensed for re-use.
+*/
+
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"aspect.build/cli/pkg/bazel"
+)
+
+var (
+	exposedBazelFlags = []string{
```

Comment:
doc this? I think "exposed" means "documented"? if so just rename

Reply:
done